_gsocprojects/2020/project_CMSSW.md

```diff
@@ -0,0 +1,12 @@
+---
+project: CMSSW
```

Comment:
Does it have a logo?

Reply:
Unfortunately it does not :(

Reply:
It can leave w/o logo, the text of CMSSW appears instead